conf/app.routes

```diff
@@ -64,16 +64,22 @@ GET         /reset-password/error
 
 GET         /applications                                                                   controllers.AddApplication.manageApps
 
-GET         /applications/add/production/10-days                                           controllers.AddApplication.tenDaysWarning
+GET         /applications/add/production/10-days                                            controllers.AddApplication.tenDaysWarning
 
-GET         /applications/add/sandbox                                                      controllers.AddApplication.addApplicationSubordinate
-GET         /applications/add/production                                                   controllers.AddApplication.addApplicationPrincipal
+GET         /applications/add/sandbox                                                       controllers.AddApplication.addApplicationSubordinate
+GET         /applications/add/production                                                    controllers.AddApplication.addApplicationPrincipal
 
-GET         /applications/add/:environment/name                                            controllers.AddApplication.addApplicationName(environment: domain.Environment)
-POST        /applications/add/:environment/name                                            controllers.AddApplication.editApplicationNameAction(environment: domain.Environment)
-GET         /applications/:id/add/:environment/subscriptions                               controllers.Subscriptions.addAppSubscriptions(id: String, environment: domain.Environment)
+GET         /applications/add/:environment/name                                             controllers.AddApplication.addApplicationName(environment: domain.Environment)
+POST        /applications/add/:environment/name                                             controllers.AddApplication.editApplicationNameAction(environment: domain.Environment)
 
-GET         /applications/:id/add/:environment/success                                     controllers.AddApplication.addApplicationSuccess(id: String, environment: domain.Environment)
+GET         /applications/:id/add/sandbox/subscription-configuration-start                  controllers.ManageSubscriptions.subscriptionConfigurationStart(id: String)
```

Comment:
could these not be rolled into fewer and use pageNumber as an optional query parameter.

Reply:
Ggiven these are new would it not be ideal to move to the newer url format discussed for api platform microservice.
`/sandbox/application/:id/subscription-configuration[?pageNumber]` (or subordinate maybe rather than sandbox).